From fa4f156c37043b2b60a75c968074a07c2bee0575 Mon Sep 17 00:00:00 2001
From: Luis Ressel <aranea@aixah.de>
Date: Mon, 13 Nov 2017 07:41:42 +0100
Subject: [PATCH] Musl compatibility

musl doesn't implement GLOB_BRACE and GLOB_TILDE, so simply don't use
them there. This only affects "setfiles -f", which I don't expect many
people use, and it's undocumented anyway that it expands globs.
---
 setfiles/restore.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/setfiles/restore.c b/setfiles/restore.c
index 50d192a..9dea565 100644
--- a/setfiles/restore.c
+++ b/setfiles/restore.c
@@ -6,6 +6,14 @@
 #include "restore.h"
 #include <glob.h>
 
+#ifndef GLOB_BRACE
+#define GLOB_BRACE 0
+#endif
+
+#ifndef GLOB_TILDE
+#define GLOB_TILDE 0
+#endif
+
 char **exclude_list;
 int exclude_count;
 
-- 
2.15.0

